theano/sandbox/cuda/basic_ops.py

```diff
@@ -121,14 +121,7 @@ class GpuFromHost(GpuOp):
 
     check_input = False
 
-    def __eq__(self, other):
-        return type(self) == type(other)
-
-    def __hash__(self):
-        return hash(type(self))
-
-    def __str__(self):
-        return 'GpuFromHost'
+    __props__ = ()
```

Comment:
You need \__props__ when the class don't have an \__init__. Don't forget the inheritance. Sometimes the parent class would have one.